content/docs/command-reference/run.md

```diff
@@ -106,6 +106,12 @@ Relevant notes:
   also means that the stage command needs to recreate any directory structures
   defined as outputs every time its executed by DVC.
 
+- In some situations we have executed a stage and later notice that output files
+  or directories created by the stage command, which are already in the
+  workspace, are missing from `dvc.yaml` (`outs` field). We can
```

Comment:
Also use the same text we have already agreed upon in content/docs/user-guide/how-to/add-output-to-stage.md please! Need to differentiate between files/dirs written by a command and formal DVC outputs. This is key... As discussed a few times.

`... later notice that some of the output files or directories it creates, which are already in the <abbr>workspace</abbr>, ...`

Reply:
Done.